scripts/shared.ts

```diff
@@ -54,10 +52,6 @@ export async function completion(
     stream
   };
 
-  if (container) {
-    bodyPayload['runner'] = { container };
-  }
-
```

Comment:
need to check if -web repo populates this field, but removed everywhere in this repo